recipes/epc/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "epc" %}
+{% set version = "0.0.5" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a14d2ea74817955a20eb00812e3a4630a132897eb4d976420240f1152c0d7d25" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: 'python setup.py install '
```

Comment:
Could this be `noarch`? Also would it be possible to use pip? https://conda-forge.org/docs/meta.html#use-pip